Modix/Services/Quote/QuoteService.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Discord;
+
+namespace Modix.Services.Quote
+{
+    public static class QuoteService
```

Comment:
Instead of having this be static, keep it instance, and register it as a Singleton with the ServiceContainer (ModixBot.cs, i think)